.gitpod.yml

```diff
@@ -9,6 +9,7 @@ tasks:
       cd packages/core
       pnpm build
       pnpm add-official-connectors
+      pnpm cli db seed
```

Comment:
I presume this connects to postgres? That will not work during prebuild as postgres isn't running then. Should be in `command` instead.